test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/jbpm/TimerIntegrationTest.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.cloud.integrationtests.jbpm;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.kie.cloud.api.DeploymentScenarioBuilderFactory;
+import org.kie.cloud.api.DeploymentScenarioBuilderFactoryLoader;
+import org.kie.cloud.api.scenario.WorkbenchWithKieServerScenario;
+import org.kie.cloud.common.logs.InstanceLogUtil;
+import org.kie.cloud.common.provider.KieServerClientProvider;
+import org.kie.cloud.git.GitProvider;
+import org.kie.cloud.git.GitProviderFactory;
+import org.kie.server.api.model.KieContainerStatus;
+import org.kie.server.api.model.KieServerInfo;
+import org.kie.server.api.model.instance.ProcessInstance;
+import org.kie.server.client.QueryServicesClient;
+import org.kie.server.integrationtests.controller.client.KieServerMgmtControllerClient;
+import org.kie.wb.test.rest.client.RestWorkbenchClient;
+import org.kie.wb.test.rest.client.WorkbenchClient;
+
+public class TimerIntegrationTest {
+
+    private static final String ORGANIZATION_UNIT_NAME = "myOrgUnit";
+    private static final String REPOSITORY_NAME = "myRepo";
+
+    private static final String PROJECT_GROUP_ID = "org.kie.server.testing";
+    private static final String PROJECT_NAME = "timer-project";
+    private static final String PROJECT_VERSION = "1.0.0.Final";
+
+    private static final String CONTAINER_ID = "cont-id";
+    private static final String CONTAINER_ALIAS = "cont-alias";
+
+    private static DeploymentScenarioBuilderFactory deploymentScenarioFactory = DeploymentScenarioBuilderFactoryLoader.getInstance();
+    private WorkbenchWithKieServerScenario workbenchWithKieServerScenario;
+    private WorkbenchClient workbenchClient;
+    private KieServerClientProvider kieServerClientProvider;
+    private KieServerMgmtControllerClient kieControllerClient;
+    private GitProvider gitProvider;
+
+    @Before
+    public void setUp() {
```

Comment:
This setup / teardown logic looks very similar to other tests in this test suite. How about consolidating it in some test util class?